### PR TITLE
Feature/add glue get job

### PR DESCRIPTION
**Pull request by malekhnovich**
* Adding boto3 get_job functionality (first time contributor so please go easy on me)

### CODE REVIEW

File: moto/glue/models.py [hunk 1]
```diff
@@ -470,5 +477,29 @@ def __init__(
...
+            "Connections": self.connections,
+            "MaxRetries": self.max_retries,
+            "AllocatedCapacity": self.allocated_capacity,
+            "Timeout": self.timeout,
+            "MaxCapacity": self.max_capacity,
+            "WorkerType": self.worker_type,
+            "NumberOfWorkers": self.number_of_workers,
+            "SecurityConfiguration": self.security_configuration,
+            "NotificationProperty": self.notification_property,
+            "GlueVesion": self.glue_version,
```
Comment (bblommers) [inline moto/glue/models.py:501]: Typo - should probably be `GlueVersion`

File: tests/test_glue/test_glue.py [hunk 2]
```diff
@@ -41,6 +42,26 @@ def test_list_jobs():
...
+    exc.value.response["Error"]["Code"].should.equal("EntityNotFoundException")
+    exc.value.response["Error"]["Message"].should.match("Job my_job_name not found")
+
+
+@mock_glue
+def test_get_job_exists():
+    client = create_glue_client()
+    job_name = create_test_job(client)
+    response = client.get_job(JobName=job_name)
+    assert response["Job"]["Name"] == job_name
```
Comment (bblommers) [inline tests/test_glue/test_glue.py:62]: Can we get a few more assertions here, just to avoid the chance of any other typo's?

**Review by bblommers — CHANGES_REQUESTED** [review]
Hi @malekhnovich, thanks for the PR! Looks good overall, just got two minor comments

File: moto/glue/models.py (hunk 1 above)
Comment (malekhnovich) [reply moto/glue/models.py]: 👍

File: tests/test_glue/test_glue.py (hunk 2 above)
Comment (malekhnovich) [reply tests/test_glue/test_glue.py:89]: yes on the way

**Review by bblommers — APPROVED** [review]
Awesome - thanks @malekhnovich!

**Pull request merged**